spec/cli/wordlist_dir_option_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
+require 'ronin/wordlists/cli/wordlist_dir_option'
+require 'ronin/wordlists/cli/command'
+
+RSpec.describe Ronin::Wordlists::CLI::WordlistDirOption do
+  class WordlistDirOptionCommandMock < Ronin::Wordlists::CLI::Command
+    include Ronin::Wordlists::CLI::WordlistDirOption
+  end
+
+  describe '.included' do
+    subject { WordlistDirOptionCommandMock }
+
+    it 'adds the --wordlist-dir option to the command' do
+      expect(subject.options).to include(:wordlist_dir)
+    end
+  end
+
+  describe '#wordlist_dir' do
+    subject { WordlistDirOptionCommandMock.new }
```

Comment:
To avoid typing the same class name over and over, I usually define a `let(:command_class) { ... }`.